Week3/README.md

```diff
@@ -20,77 +20,93 @@ Your teachers Igor and Unmesh have made video lectures for this week's material.
-| Apple        | Macbook Pro | United States        |
-| Lenovo       | ThinkPad    | China                |
-| Lenovo       | IdeaPad     | China                |
+This table violates the rule of 2NF and 3NF because data in the column `Manufacturer country` and `Manufacturer country code` depend on the Primary Key that is actually unrelated.
+To make the database comply, we have to split up the table into three smaller tables (Countries, Models, Manufacturers):
 
-This table violates the rule of 2NF because data in the column `Manufacturer country` depends on the `Manufacturer` column (which can be considered a subset of the key {`Manufacturer`,`Model`}). To make that database comply with the 2NF, we have to split up the table into two smaller tables:
+**Countries**
 
-| Manufacturer | Manufacturer country |
-| ------------ | -------------------- |
-| Apple        | United States        |
-| Lenovo       | China                |
+| Country Code | Country Name  |
+| ------------ | ------------- |
+| US           | United States |
+| CN           | China         |
 
-| Manufacturer | Model       |
-| ------------ | ----------- |
-| Apple        | MacBook Air |
-| Apple        | Macbook Pro |
-| Lenovo       | ThinkPad    |
-| Lenovo       | IdeaPad     |
+**Models**
 
-#### 3rd normal form (3NF)
+| Model       | Manufacturer |
+| ----------- | ------------ |
+| MacBook Air | Apple        |
+| Macbook Pro | Apple        |
+| ThinkPad    | Lenovo       |
+| IdeaPad     | Lenovo       |
 
-There is only one rule for third normal form: No transitive dependency (i.e. no field should depend on non-key attributes).
+**Manufacturers**
 
-This rule is very similar to the rule for 2NF with the exception that it applies to non-key attributes. The approach for making your database comply with 3NF is also similar: split up your table. You can find a complete example [here](https://en.wikipedia.org/wiki/Third_normal_form).
+| Manufacturer | Country Code |
+| ------------ | ------------ |
+| Apple        | US           |
+| Lenovo       | CN           |
+
+Update anomalies cannot occur in these tables, unlike before because there is only one value for country name for each country code.
 
 #### Higher normal forms
 
-Besides these three normal forms, there are other higher normal forms, for example, the Boyce-Codd normal form (3.5 NF). Not all of these normal forms are equally important. Most of the time while designing databases you should aim for the third normal form. If you still did not get what are the rules for thedifferentt normal forms, don't worry. Over time you will develop a feel for it.
+Besides these three normal forms, there are other higher normal forms, for example, the Boyce-Codd normal form (3.5 NF).
+Not all of these normal forms are equally important.
+Most of the time while designing databases you should aim for the third normal form.
+If you still did not get what are the rules for the different normal forms, don't worry.
```

Comment:
The lines 106 and 107 are SUPER important. Thanks for adding them.

Reply:
I'm not going to take the credit. This was already in the readme 😊
I just added line breaks because this line was way to long